### PR TITLE
Update build instruction

### PR DESCRIPTION
**Pull request by ctwo1618**
Update build instructions.

### CODE REVIEW

File: BUILD.md [hunk 1]
```diff
@@ -10,13 +10,14 @@ git clone https://github.com/dm-vdo/kvdo
 
 Build the uds library:
 ~~~
-make -C $BUILD_DIR/vdo/utils/uds/
+cd   $BUILD_DIR/kvdo
+make -C /usr/src/kernels/`uname -r` M=`pwd`
```
Comment (johnwiele) [inline BUILD.md]: It is necessary to build the user space UDS library. It is totally unnecessary to build the kvdo kernel module. The only reason kvdo is needed is to provide block.c, which for some reason is not in the user space library.

File: BUILD.md [hunk 2]
```diff
@@ -10,13 +10,14 @@ git clone https://github.com/dm-vdo/kvdo
...
 ~~~
 
 Get and build the LZ4 library:
 ~~~
 git clone https://github.com/lz4/lz4
```
Comment (johnwiele) [inline BUILD.md]: After the "cd" above, the current directory is $BUILD_DIR/kvdo so this will put lz4 there.

File: BUILD.md [hunk 3]
```diff
@@ -10,13 +10,14 @@ git clone https://github.com/dm-vdo/kvdo
...
-make -C $BUILD_DIR/lz4/lib/
+make -C $BUILD_DIR/lz4/ lib
```
Comment (johnwiele) [inline BUILD.md]: What is the advantage of building the "lib" target in lz4 as opposed to building the default target of lz4/lib?

**Comment by ctwo1618** [comment]
The original "make -C $BUILD_DIR/lz4/lib/" does not work.  From that line,
I figure you just want to build the library

Chung

**Comment by ctwo1618** [comment]
I figure the instruction is to do all the git in the BUILD_DIR.

Chung

**Comment by ctwo1618** [comment]
the main thing is

-make -C $BUILD_DIR/vdo/utils/uds/

Does not work.  If one want to build the UDS library, we will to do this:
make -C /usr/src/kernels/`uname -r` M=`pwd`

It is in the kvdo REDEME file.

Chung

**Comment by ctwo1618** [comment]
My misread of instruction, request pull.

**Pull request closed without merging**